charter.md

```diff
@@ -2,47 +2,33 @@
-
-In order to meet privacy, security, and sustainability objectives, digital credentials need to be designed with awareness of computation and storage constraints associated with their use cases.
-The SPICE WG focuses on a few explicit higher level objectives: clear semantics, proof schemes supporting both traceability and unlinkability, data minimization and selective disclosure.
-
-These objectives can be achieved by leveraging industry adopted standards and managing trade-offs between cutting-edge and well-established cryptography.
-
-As a guiding principle for the SPICE WG, claims that work well in JSON must work well in CBOR, without loss of semantics. Compact expressions of claims consume less resources and expose less attack surface, these properties are in support sustainable design.
-
-The SPICE WG will support digital credential formats leveraging existing IETF standards and IANA registries, and create a framework for consistently extending them to support stakeholders that are building compliance and automation systems based on industry adopted cryptography and protocols.
-
-Digital credentials are identity documents with attributes (public or private claims in IANA registries) about the identity. The identifiers referenced could be about any subject, but common examples include people, devices, assets, organizations and processes.
+Digital credentials are essential to identity, access control, and digitization use cases that are part of modernization efforts, such as digital licenses or certificates of origin.
+COSE and JOSE provide building blocks, such as identifiers and public and private claim structures, which enable interoperability and cryptographic agility, but which require profiling to render digitization easy, unambiguous and consistent.
+There are various ways to combine identifiers, key material, signatures, and attributes to create credentials, which often prevents interoperabtility from spanning across data silos or cross use cases.
+This lack of interoperabilty leads to increased implementation costs, attack surface, and harms adoption, which prevents digitzation from being deployed successfully.
+SPICE aligns the capabilities of JOSE and COSE to support modern cryptographic capabilities, such as selective disclosure or unlinkablilty, consistent identity attribute semantics across representations, and Internet-scale discoverability of identifiers and cryptography capabilities.
 
 ## Background
 
-During the last few years there has been renewed interest in privacy-enhancing technologies for identity management systems that utilize technologies offering selective disclosure, data minimization and unlinkability.
+Foundational building blocks have been developed with BBS Signatures, RSA Blind Signatures, Verifiable Random Functions, or other Selective-Disclosure and data minimization techniques.
```

Comment:
Developed by whom/where?

Reply:
I think in a charter, we don't need references for these but all of the listed ones here have been developed at CFRG or Privacy Pass or OAuth.